src/Card/README.md

```diff
@@ -33,7 +33,8 @@ This component uses a `Card` from react-bootstrap as a base component and extend
 
   return (
     <Card style={{ width: isExtraSmall ? "100%" : "18rem" }}>
-      <Card.ImageCap 
+      <Card.ImageCap
+        skeletonHeight={140}
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` prop here?

Reply:
Removed